go/arrow/cdata/cdata.go

```diff
@@ -683,6 +743,43 @@ func (imp *cimporter) importListLike() (err error) {
 	return
 }
 
+func (imp *cimporter) importListViewLike() (err error) {
+	offsetSize := int64(imp.dt.Layout().Buffers[1].ByteWidth)
+
+	if err = imp.checkNumChildren(1); err != nil {
+		return err
+	}
+
+	if err = imp.checkNumBuffers(3); err != nil {
+		return err
+	}
+
+	var nulls, offsets, sizes *memory.Buffer
+	if nulls, err = imp.importNullBitmap(0); err != nil {
+		return
+	}
+	if nulls != nil {
+		defer nulls.Release()
+	}
+
+	if offsets, err = imp.importOffsetsBuffer(1, offsetSize); err != nil {
```

Comment:
Is `importOffsetsBuffer` importing only `len` offsets instead of `len + 1`?

Reply:
Nope, will replace with importFixedSizeBuffer